index.fr.md

```diff
@@ -0,0 +1,147 @@
+    Vous pouvez utiliser les images de cette page pour l’éducation à l’accessibilité et sa sensibilisation si vous&nbsp;:<br>
+    1. Incluez l’URI <strong><span class="changed">w3.org/WAI/fundamentals/components/</span> <em>en évidence</em></strong> à proximité de l’image, et <br>
+    2. Incluez l’attribution à l’artiste, l’éditeur, et la mention de copyright dans tous les travaux publiés ou diffusés&nbsp;:<br><cite>Image par Michael Duffy, depuis&nbsp;: Essential Components of Web  Accessibility. Éditrice S. L. Henry. Copyright W3C <sup>®</sup> (MIT, ERCIM, Keio, Beihang). w3.org/WAI/fundamentals/components/</cite><br>
+    Pour plus d’informations, voir <a href="https://www.w3.org/WAI/about/usingWAImaterial.html" lang="en" hreflang="en">Using WAI Materials (en anglais)</a>.
+  </p>
+  <p><strong>Date&nbsp;: </strong>Mis à jour le 27 février 2018. <!-- [<a href="@@">Changelog</a>] --> </p>
+  <p><strong>Éditrice&nbsp;:</strong> <a href="http://www.w3.org/People/Shawn" lang="en" hreflang="en">Shawn Lawton Henry</a>. Graphiste&nbsp;: Michael Duffy.</p>
+  <p>Élaboré avec des contributions du groupe de travail «&nbsp;Éducation et Sensibilisation&nbsp;» (<span lang=”en”>Education and Outreach Working Group</span>) (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+ 
+ 
+# Read Important Translations Guidance at https://www.w3.org/WAI/about/translating/#important
+# Read Translations Notes for this resource at https://github.com/w3c/wai-components/blob/master/README.md
+# end of translation instructions 
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Résumé" class="full" %}
+{:/}
+ 
+Cette page montre comment l’accessibilité dépend de l’articulation harmonieuse de nombreuses composantes, et comment des améliorations de certaines pourraient améliorer sensiblement l’accessibilité du web.
+ 
+Elle fournit les bases pour comprendre les différents standards d’accessibilité produits par l’Initiative pour l’Accessibilité du Web (<span lang="en">Web Accessibility Initiative WAI</span>) du <abbr title="World Wide Web Consortium">W3C</abbr>.
+ 
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+ 
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Contenu de la page" class="simple" %}
+{:/}
+ 
+{::options toc_levels="2" /}
+ 
+-   This text will be replaced by the TOC.
+{:toc}
+ 
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+ 
+ 
+## Introduction {#intro}
+{:.no_toc}
+ 
+Il est essentiel que des composantes distinctes et multiples du développement web et de l’interaction utilisateur s’articulent convenablement pour que le Web soit accessible aux personnes handicapées. Ces composantes incluent&nbsp;:
+ 
+-   **du contenu** – l’information contenue dans une page web ou une application web, y compris&nbsp;:
+	-   de l’information primaire telle que du texte, des images et des sons,
+	-   du code ou du balisage qui définit la structure, la présentation… &nbsp;;
```

Comment:
@3l3gant-cod3s Do you see a reason to use suspension points rather than `etc.` commonly used in French?

```suggestion
	-   du code ou du balisage qui définit la structure, la présentation, etc.&nbsp;;
```

Reply:
I prefer the shorter version but I can live with it… ;)

Reply:
cf https://fr.wikipedia.org/wiki/Points_de_suspension

> Ils peuvent aussi être utilisés : (…)
> * à la fin de listes non exhaustives : « … » a la même valeur que « , [etc.](https://fr.wikipedia.org/wiki/Etc.) »

Reply:
@3l3gant-cod3s Feel free to use the version you are the most comfortable with!